kf_lib_data_ingest/common/constants.py

```diff
@@ -286,6 +286,10 @@ class UCSF_CAT:
             NAME = "UCSF CAT"
             KF_ID = "SC_MSQP04K7"
 
+        class UPENN:
+            NAME = "University of Penn"
```

Comment:
```suggestion
            NAME = "University of Pennsylvania"
```

Unless there's some context I'm missing, shouldn't this be `Pennsylvania` instead of `Penn`?

Reply:
oh, yeah. have updated.